src/sql/src/rbac.rs

```diff
@@ -909,6 +909,13 @@ fn generate_rbac_requirements(
             ownership: vec![ObjectId::Item(*id_a), ObjectId::Item(*id_b)],
             ..Default::default()
         },
+        Plan::AlterSchemaRename(plan::AlterSchemaRenamePlan {
+            cur_schema_spec,
+            new_schema_name: _,
+        }) => RbacRequirements {
+            ownership: vec![ObjectId::Schema(*cur_schema_spec)],
+            ..Default::default()
```

Comment:
From the PostgreSQL docs (https://www.postgresql.org/docs/16/sql-alterschema.html)
> You must own the schema to use `ALTER SCHEMA`. To rename a schema you must also have the `CREATE` privilege for the database.

Reply:
You're missing the `CREATE` privilege here.

Reply:
Updated! I didn't know what permission to use for schemas in the ambient database, so I kind of guessed.